src/components/LoadingCard/LoadingCard.css

```diff
@@ -0,0 +1,33 @@
+.loading {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index:0;
+    border-radius: 15px; /* Sync with Card */
+}
+
+.loading::before {
+    content:"Loading...";
+}
+
+@media screen and (prefers-reduced-motion: no-preference) and (forced-colors: none) {
+    .loading:before {
+        content:"";
+        position:absolute;
+        z-index:-1;
+        top:0;
+        left:0;
+        right:-100%;
+        bottom:0;
+        animation: aaa 5s linear infinite;
+        background-size: 200%;
+        background: linear-gradient(0.25turn, #eee, #fff, #eee) left/50% 100%;
+    }
+}
+
+@keyframes aaa {
```

Comment:
aaa

Reply:
What should I name this? It's just a local animation and "loading" is taken

Reply:
`animation` or something